metabase/metabase
# Mantine color and typography fixes

Fixes colors and line heights across the components.

How to verify:
- `yarn storybook`
- Go over each mantine component, open the figma file, and make sure they match

<img width="1064" alt="Screenshot 2023-08-24 at 16 55 19" src="https://github.com/metabase/metabase/assets/8542534/4fce50b4-ac65-4824-8f73-74eec4439b01">
